node/src/actors/chain_manager/mod.rs

```diff
@@ -936,7 +942,7 @@ where
 fn update_reputation(
     rep_eng: &mut ReputationEngine,
     consensus_constants: &ConsensusConstants,
-    miner_pkh: PublicKeyHash,
+    miner_pkh: Option<PublicKeyHash>,
```

Comment:
This should not be an option, and it should be the pkh used in the VRF proof, not the one used in the mint.